runner_manager/models/backend.py

```diff
@@ -116,3 +116,60 @@ def configure_instance(self, runner: Runner) -> Instance:
             labels=self.labels,
             metadata=Metadata(items=items),
         )
+
+
+class AWSConfig(BackendConfig):
+    """Configuration for AWS backend."""
+
+    region: str = "us-west-2"
+    subnet_id: str
+
+
+class AWSInstanceConfig(InstanceConfig):
+    """Configuration for AWS backend instance."""
+
+    image: str = "ami-0735c191cf914754d"  # Ubuntu 22.04 for us-west-2
```

Comment:
Need to make sure this is a gp3 AMI for the disks

Reply:
I think its not tied to the ami you can set that while creating the volume

Reply:
OK nice